### PR TITLE
Update embed.md

### LINKED ISSUE #3601: update embed.md doc with limitations

### PR DESCRIPTION
**Pull request by minniewong**
## Description

Fixes #3601  .
 
Updated the embed doc to add more information. Will have to add a link to an Earthdata webpage with examples when embed mode is live. 

[If this is a relatively large or complex change, start a discussion by explaining why you chose the solution you did and what alternatives you considered, etc...]

## How To Test

[Provide whatever information a reviewer might need to know in order to verify that the changes made in this PR are working as expected. If there are special build steps that need to be taken in order to get these changes to run (building while on a separate branch, running `npm ci`, etc) include them here.]

- For bugfixes: What steps need to be taken in the UI to verify the bug is fixed?
- For enhancements and features: What is the expected functionality being added/modified? How can a reviewer verify this?

1. Open with these URL parameters
2. ...
3. Verify expected result

## PR Submission Checklist

This is simply a reminder of what we are going to look for before merging your code.

- I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/main/.github/CONTRIBUTING.md) doc
- I have added necessary documentation (if applicable)
- I have added tests that prove my fix is effective or that my feature works (if applicable)
- Any dependent changes have been merged and published in downstream modules (if applicable)

@nasa-gibs/worldview

### CODE REVIEW

File: doc/embed.md [hunk 1]
```diff
@@ -1,11 +1,35 @@
...
+3. Click on the Share icon (second from the left) in the upper right corner.
+4. Click on the "Embed" tab.
+5. Click "Copy" to copy the HTML code to wrap the current Worldview instance in an `<iframe>` inline element.
```
Comment (edplato) [inline doc/embed.md:9]:
```suggestion
5. Click "Copy" to copy the HTML code with the current Worldview instance in an embeddable `<iframe>` inline element.
```

File: doc/embed.md [hunk 2]
```diff
@@ -1,11 +1,35 @@
...
 
 **Example `<iframe>` code:**
 
 ```<iframe src="https://worldview.earthdata.nasa.gov/?em=true" role="application" sandbox="allow-modals allow-scripts allow-same-origin allow-forms allow-popups" width="100%" height="100%" allow="fullscreen; autoplay;" loading="lazy"></iframe>```
 
-**Note:** Certain application features and button/tab interactions are not available in embed mode. We recommend to limit the number of layers you share - hidden layers will be filtered out and are not toggleable by the end user. A helpful tip is to provide detailed captions to assist the user in better understanding the science concepts that you are sharing.
+**Notes:** Compared to the full application, the embedded version of Worldview has intentionally limited functionality to give users a more streamlined experience. The following is a list of the features and button/tab interactions that are not available or altered in embed mode.
+
+- Search places by location, Share this map, Switch projections, Take a snapshot, and the Information button are disabled. These features have been replaced with a button that brings user back to the full version of Worldview showing the embedded map contents.
```
Comment (edplato) [inline doc/embed.md:21]:
```suggestion
- Search places by location, Share this map, Switch projections, Take a snapshot, and Information buttons are disabled. These features have been replaced with a button that opens up a new tab with your current map view in the full featured Worldview.
```

File: doc/embed.md [hunk 3]
```diff
@@ -1,11 +1,35 @@
...
+- The welcome page and its tour stories are disabled.
+- In the Layer List, layer grouping is disabled, the hidden layers are not displayed, and the options to change the opacity, thresholds, color palettes, disable/enable classifications, or other related settings are not available.
+- The Events tab will only appear in the embedded version if it is selected in Worldview before creating the iframe html code.
```
Comment (edplato) [inline doc/embed.md:24]:
```suggestion
- The Events tab will only appear in the embedded version if both 1) the tab is selected and 2) an event is selected in Worldview before creating the `iframe` html code.
```

File: doc/embed.md [hunk 4]
```diff
@@ -1,11 +1,35 @@
...
+- The Events tab will only show the event that was initially selected in Worldview.
```
Comment (edplato) [inline doc/embed.md:25]:
```suggestion
```

File: doc/embed.md [hunk 5]
```diff
@@ -1,11 +1,35 @@
...
+- Event tab options are disabled.
```
Comment (edplato) [inline doc/embed.md:26]:
```suggestion
```

File: doc/embed.md [hunk 6]
```diff
@@ -1,11 +1,35 @@
 # Embedding Worldview
```
Comment (edplato) [inline doc/embed.md:2]:
```suggestion

## Create an Embedded Worldview
```

File: doc/embed.md [hunk 7]
```diff
@@ -1,11 +1,35 @@
...
+3. Click on the Share icon (second from the left) in the upper right corner.
+4. Click on the "Embed" tab.
+5. Click "Copy" to copy the HTML code to wrap the current Worldview instance in an `<iframe>` inline element.
+6. Add the HTML code to your webpage in the place where you want the Worldview map to appear.
+Note: You will need to wrap the `<iframe>` code in a wrapper element (e.g., `<div>`) with your desired width, height, and any additional style changes. You can change the width and height from percent to px to make it fit your page.
```
Comment (edplato) [inline doc/embed.md:11]:
```suggestion
Note: You will need to wrap the `<iframe>` code in a wrapper element (e.g., `<div>`) with your desired width, height, and any additional style changes. You can change the width and height from percent to px to make it fit your page. See [https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe) for more information on the `<iframe>` inline element including an explanation of its attributes.
```

File: doc/embed.md [hunk 8]
```diff
@@ -1,11 +1,35 @@
...
 
 See [https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe) for more information on the `<iframe>` inline element including an explanation of its attributes.
```
Comment (edplato) [inline doc/embed.md:13]:
```suggestion
```

File: doc/embed.md [hunk 9]
```diff
@@ -1,11 +1,35 @@
...
 
 **Example `<iframe>` code:**
```
Comment (edplato) [inline doc/embed.md:16]:
```suggestion
```

File: doc/embed.md [hunk 10]
```diff
@@ -1,11 +1,35 @@
...
 
 ```<iframe src="https://worldview.earthdata.nasa.gov/?em=true" role="application" sandbox="allow-modals allow-scripts allow-same-origin allow-forms allow-popups" width="100%" height="100%" allow="fullscreen; autoplay;" loading="lazy"></iframe>```
 
-**Note:** Certain application features and button/tab interactions are not available in embed mode. We recommend to limit the number of layers you share - hidden layers will be filtered out and are not toggleable by the end user. A helpful tip is to provide detailed captions to assist the user in better understanding the science concepts that you are sharing.
+**Notes:** Compared to the full application, the embedded version of Worldview has intentionally limited functionality to give users a more streamlined experience. The following is a list of the features and button/tab interactions that are not available or altered in embed mode.
```
Comment (edplato) [inline doc/embed.md:19]:
```suggestion
## Limited Features in an Embedded Worldview

Compared to the full application, the embedded version of Worldview has intentionally limited functionality to give users a more streamlined experience. The following is a list of the features and button/tab interactions that are not available or altered in embed mode.
```

File: doc/embed.md [hunk 11]
```diff
@@ -1,11 +1,35 @@
...
+- In the Layer List, layer grouping is disabled, the hidden layers are not displayed, and the options to change the opacity, thresholds, color palettes, disable/enable classifications, or other related settings are not available.
+- The Events tab will only appear in the embedded version if it is selected in Worldview before creating the iframe html code.
+- The Events tab will only show the event that was initially selected in Worldview.
+- Event tab options are disabled.
+- Data Download is disabled.
+- The Extended timeline and layer coverage panel are unavailable. However, it is still possible for users to change the date.
+- The Export to Animation tool is disabled.
+- The Measure Tool is disabled.
+
+Helpful Tips when embedding:
```
Comment (edplato) [inline doc/embed.md:32]:
```suggestion
## Helpful Tips

```

**Review by edplato — APPROVED** [review]
I made some formatting/content changes. You may need to tweak since the suggested changes aren't showing a preview so there may be wonky formatting.

File: doc/embed.md (hunk 10 above)
Comment (rboller) [reply doc/embed.md:19]: `The following is a list of the features and button/tab interactions that are not available or altered in embed mode.` -- I'd suggest replacing the `.` with a `:`.

**Review by rboller — APPROVED** [review]
I made one small grammatical tweak and otherwise Ed had my suggestions covered.

File: doc/embed.md (hunk 4 above)
Comment (minniewong) [reply doc/embed.md:25]: ✔️

**Pull request merged**